project/plugins.sbt

```diff
@@ -1 +1,3 @@
 addSbtPlugin("com.eed3si9n" % "sbt-doge" % "0.1.5")
+
+addSbtPlugin("de.johoop" % "sbt-testng-plugin" % "3.1.1")
```

Comment:
Why is this needed?

Reply:
Without this, the TestNG-based tests weren't running in sbt. For Scala 2.11, I had to update from 3.0.2 to 3.1.1 to get it to build.

Reply:
I just tried to remove it and it works. This `addSbtPlugin` is adding dependency for the library projects' `build.sbt`. Since the library projects (NGPlugin and NGHelper) do not have any TestNG tests, they do not need this in their project settings.